transcom/mymove
# B-21364 I-12546 INT

## [Agility Issue](https://www13.v1host.com/USTRANSCOM38/Issue.mvc/Summary?oidToken=Issue%3A1033795)
## [Agility ticket](https://www13.v1host.com/USTRANSCOM38/assetdetail.v1?number=B-21364)

## Summary

- Added a check for UB shipment types in src/pages/MyMove/Multi-Moves/MultiMovesMoveContainer/MultiMovesMoveContainer.jsx, to display "Unaccompanied Baggage" rather than the unspecified "Shipment"
- Updated tests

### How to test

1. Access the milmove app as a customer
2. Create a UB shipment
3. After creating, head to the move home page and expand the shipments drop down.
4. Verify that you see Unaccompanied Baggage" on the page.

### Frontend

![image](https://github.com/user-attachments/assets/a64704ef-8f49-4d69-8fe6-fc0803ac7889)

## Screenshots
Before:
![image](https://github.com/user-attachments/assets/612382ec-6f8b-4947-9441-bcc045891203)

After:
![image](https://github.com/user-attachments/assets/baba68e5-93b9-4042-a8a1-bf9f2d5476d8)
